tasks/UsblDock.cpp

```diff
@@ -29,14 +29,11 @@ bool UsblDock::configureHook()
     if (! UsblDockBase::configureHook())
         return false;
 
-    if (!_io_port.get().empty())
+    if(_interface.get() != ETHERNET)
```

Comment:
If you FORCE the property to be a certain value, then don't make it a property !